.github/workflows/publish.yml

```diff
@@ -0,0 +1,261 @@
+      image_tag: ${{ steps.push_docker_image.outputs.image_tag }}
+      release_name: ${{ steps.push_docker_image.outputs.release_name }}
+      chart_version: ${{ steps.publish_helm_chart.outputs.chart_version }}
+
+    env:
+      TEST_COVERAGE_FAIL_THRESHOLD: 45
+      TEST_COVERAGE_WARNING_THRESHOLD: 60
+      COMMIT_URL: ${{github.event.head_commit.url}}
+      COMMITTER: ${{github.event.head_commit.committer.name}}
+
+    steps:
+      - uses: azure/setup-helm@v3
+        with:
+          version: '3.12.2'
+        id: install
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 16.x
+          cache: 'yarn'
+
+      - name: Set environment variables
+        run: |
+          echo "BRANCH_NAME=$(git branch --show-current)" >> $GITHUB_ENV
+          echo "TAG=$(git tag --points-at ${{github.sha}})" >> $GITHUB_ENV
+          echo "PR_LAST_COMMIT_MESSAGE=$(git log --format=%B -n 1 ${{github.event.after}} | head -n 1)" >> $GITHUB_ENV
+
+      - uses: kanga333/variable-mapper@master
+        name: Determine target branches in wireapp/wire-builds
+        with:
+          key: '${{env.BRANCH_NAME}}'
+          map: |
+            {
+              "stefan": { "wire_builds_target_branches" : "[\"main\"]" },
+              "dev": { "wire_builds_target_branches" : "[\"main\"]" }
+            }
+
+      - name: Make target branches available as output to other jobs
+        id: output_target_branches
+        shell: bash
+        run: |
+          echo "wire_builds_target_branches: $wire_builds_target_branches"
+          echo "wire_builds_target_branches=$wire_builds_target_branches" >> $GITHUB_OUTPUT
+
+      - name: Set TITLE and BUILD_DESKTOP
+        env:
+          PR_TITLE: ${{github.event.pull_request.title || env.PR_LAST_COMMIT_MESSAGE}}
+        run: |
+          echo "TITLE=${PR_TITLE}" >> $GITHUB_ENV
+          echo "BUILD_DESKTOP=${{contains(env.TAG, 'staging') || contains(env.TAG, 'production') || contains(env.PR_LAST_COMMIT_MESSAGE, '+Desktop')}}" >> $GITHUB_ENV
+
+      - name: Print environment variables
+        run: |
+          echo -e "BRANCH_NAME = ${BRANCH_NAME}"
+          echo -e "TAG = ${TAG}"
+          echo -e "TITLE = ${TITLE}"
+          echo -e "PR_LAST_COMMIT_MESSAGE = ${PR_LAST_COMMIT_MESSAGE}"
+          echo -e "COMMIT_URL = ${COMMIT_URL}"
+          echo -e "COMMITTER = ${COMMITTER}"
+          echo -e "BUILD_DESKTOP = ${BUILD_DESKTOP}"
+
+      - name: Skip CI
+        if: |
+          contains(env.TITLE || env.PR_LAST_COMMIT_MESSAGE, 'skip ci') ||
+          contains(env.TITLE || env.PR_LAST_COMMIT_MESSAGE, '[ci skip]')
+        uses: andymckay/cancel-action@0.3
+
+      - name: Authenticate git clone
+        env:
+          GH_TOKEN: ${{secrets.OTTO_THE_BOT_GH_TOKEN}}
+        run: echo "machine github.com login ${GH_TOKEN}" > ~/.netrc
```

Comment:
question: do you need to be an authenticated user for this script to run properly? I don't think we need (and actually our other job might not even need it either 🤔 )

Reply:
Yeah, it wasn't needed. I removed it from both workflows in 119af02dcf04069e8f4cde240fd8e0872547834c and everything succeeded